tools/swift-machine-learning-api/codes/trainer/input_pipeline_dask.py

```diff
@@ -0,0 +1,521 @@
+# Copyright 2019 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""
+Input Pipeline for loading data into the model
+"""
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+from multiprocessing import cpu_count
+
+from google.cloud import storage
+import numpy as np
+import dask.dataframe as dd
+import dask
+import tensorflow as tf
+import lime.lime_tabular
+import dill
+import pandas as pd
+import os
+
+
+class InputReader(object):
+    """Class for reading input from different sources
+    Assuming csv files for now
+    """
+
+    def __init__(self, csv_path, task_type, target_var, na_values=None,
+                 column_names=None,
+                 to_drop=None, gcs_path=False, data_type=None):
+        """The init method initialise and keeps track of the source of input
+        (say csv, json etc) and other variables.
+
+        Arguments:
+                csv_path : string, Path of the csv files whether local or on remote storage
```

Comment:
please run pylint and respect 80 char line length.

Reply:
Modified.